internal/scraper/scraper.go

```diff
@@ -122,28 +129,31 @@ func New(
 	labelsLimiter LabelsLimiter,
 	telemeter *telemetry.Telemeter,
 	secretStore *secrets.TenantSecrets,
+	cals *cals.CostAttributionLabels,
```

Comment:
You can use the interface here as well:

```suggestion
	cals TenantCals,
```